### ISSUE TITLE
.NET 5 not recognized on Arch Linux

### ISSUE THREAD

**Issue by CoralCunt** [Pathoschild/SMAPI#817]
SMAPI does not start despite .NET 5 being installed on Arch. When you try to start Stardew on either Steam or Lutris, the button will briefly show that it's running, and then return to the regular play button. 

I ran Steam in the terminal to get some info. It returned this: 
````
GameAction [AppID 413150, ActionID 2] : LaunchApp changed task to Completed with ""
pid 226725 != 226722, skipping destruction (fork without exec?)
pid 226955 != 226950, skipping destruction (fork without exec?)
Oops! You must have .NET 5 installed to use SMAPI: https://dotnet.microsoft.com/download
````

I downloaded .NET 5 based on [this guide](https://wiki.archlinux.org/title/.NET#Install_multiple_versions_via_AUR) on the Arch Wiki. 

I installed the following packages: 
- dotnet-host
- dotnet-runtime-5.0
- dotnet-sdk-5.0

**To Reproduce**
1. Run the "Install on Linux.sh" file to install SMAPI
2. Click play on a launcher of your choice
3. Nothing happens

**Log file**
No logs, as SMAPI can't seem to be able to create one

Note: Previously when running the installer, it cancelled the installation because I lacked .NET 5. After installing dotnet-runtime-5.0, it ran just fine, but running the game itself does not work despite this. I did not have this issue on SMAPI 12.X .

**Comment by CoralCunt** [comment]
Update: A workaround is adding the StardewValley shell script as a native Linux game executable in Lutris and playing that way. I was previously using StardewModdingAPI.bin.x86_64 as the executable. This seems to be broken now. 

This issue still seems like something that needs to be addressed, so I'll keep it open.

**Comment by Pathoschild** [comment]
Fixed in SMAPI 3.13.2, which no longer needs any version of .NET to be installed. Thanks for reporting it!
